source/manual/business-readiness-publish-changes.html.md

```diff
@@ -31,15 +31,15 @@ The content item for the finder is updated from a [YAML file][govuk-app-deployme
 
 ## Updating the business readiness finder
 
-The content item for the business readiness finder and its email signup page are published by `search-api`. However the actual config is read in from [govuk-app-deployment-secrets][govuk-app-deployment-secrets], and config for the facet group is managed in [content-tagger][content-tagger].
+The content item for the business readiness finder and its email signup page are published by `search-api` and the config for the facet group is managed in [content-tagger][content-tagger].
 
 1. Update the facet group in [content-tagger][content-tagger]
 2. Merge the changes to [content-tagger][content-tagger]
 3. Deploy content-tagger
 4. [Import and Publish the facet group](/manual/publishing-a-facet-group.html#how-to-import-or-update-a-facet-group)
-5. Update [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
-6. Merge the changes to [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
-7. Re-deploy `search-api` so it can pick up the changes in [govuk-app-deployment-secrets][govuk-app-deployment-secrets]
+5. Update [search-api][search-api]
```

Comment:
These update and merge steps don't make sense to me. They feel a bit dependent on the type of change that's being made, perhaps this could be made clearer. Not necessary for this PR as it's just updating details about which apps handle specific steps.

Reply:
You're right, but the steps need a bit more thought.